NitroxServer/UnityStubs/Vector3.cs

```diff
@@ -5,6 +5,7 @@ namespace NitroxServer.UnityStubs
     [ProtoContract]
     public class Vector3
     {
+#pragma warning disable IDE1006 // Naming Styles
```

Comment:
I wish IDE supported built in ignore list mastered in a new file that didn't muddy up the code with pragma warning disables.

Reply:
Interesting, I cannot remember adding these manually on purpose. I can alter the `.editorconfig` to stop complaining about naming, because that's where this is coming from.